drivers/SmartThings/matter-appliance/src/matter-refrigerator/init.lua

```diff
@@ -96,15 +96,43 @@ local function is_matter_refrigerator(opts, driver, device)
   return false
 end
 
-local function temperature_setpoint_attr_handler(driver, device, ib, response)
-  local tn_eps = embedded_cluster_utils.get_endpoints(device, clusters.TemperatureControl.ID, {feature_bitmap = clusters.TemperatureControl.types.Feature.TEMPERATURE_NUMBER})
+local function supports_temperature_level_endpoint(device, endpoint)
+  local feature = clusters.TemperatureControl.types.Feature.TEMPERATURE_LEVEL
+  local tl_eps = embedded_cluster_utils.get_endpoints(device, clusters.TemperatureControl.ID, {feature_bitmap = feature})
+  if #tl_eps == 0 then
+    device.log.warn_with({ hub_logs = true }, string.format("Device does not support TEMPERATURE_LEVEL feature"))
+    return false
+  end
+  for i, eps in ipairs(tl_eps) do
+    if eps == endpoint then
+      return true
+    end
+  end
+  device.log.warn_with({ hub_logs = true }, string.format("Endpoint(%d) does not support TEMPERATURE_LEVEL feature", endpoint))
+  return false
+end
+
+local function supports_temperature_number_endpoint(device, endpoint)
+  local feature = clusters.TemperatureControl.types.Feature.TEMPERATURE_NUMBER
+  local tn_eps = embedded_cluster_utils.get_endpoints(device, clusters.TemperatureControl.ID, {feature_bitmap = feature})
   if #tn_eps == 0 then
     device.log.warn_with({ hub_logs = true }, string.format("Device does not support TEMPERATURE_NUMBER feature"))
-    return
+    return false
   end
-  device.log.info_with({ hub_logs = true },
-    string.format("temperature_setpoint_attr_handler: %d", ib.data.value))
+  for i, eps in ipairs(tn_eps) do
+    if eps == endpoint then
+      return true
+    end
+  end
+  device.log.warn_with({ hub_logs = true }, string.format("Endpoint(%d) does not support TEMPERATURE_NUMBER feature", endpoint))
```

Comment:
There are a lot of cases where we are logging to the hubLogs throughout this file. Could some of these be removed or isolated to the driver logs (by using just `device.log.info()` instead of `device.log.info_with({hub_logs = true}, ..)`) so that we do not clutter the hub logs with unnecessary information?

Reply:
Okay I'll update it :)

Reply:
It is the same with [the comment](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1500/files#r1694798795) above